src/Catty/ViewController/Help/HelpWebViewController.swift

```diff
@@ -30,27 +32,33 @@ class HelpWebViewController: UIViewController, UIWebViewDelegate, UIScrollViewDe
     private var url: URL?
     private var touchHelperView: UIView?
     private var loadingView: LoadingView?
+    private var webView: WKWebView?
 
     @IBOutlet private weak var urlTitleLabel: UILabel!
     @IBOutlet private weak var progressView: UIProgressView!
-    @IBOutlet private weak var webView: UIWebView!
-
-    // MARK: - Init
-
-    required init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
-    }
-
-    deinit {
-        url = nil
-        webView.delegate = nil
-        webView = nil
-    }
+    @IBOutlet private weak var viewForWebView: UIView!
 
     // MARK: - View Lifecycle
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        let webConfiguration = WKWebViewConfiguration()
+        webConfiguration.allowsInlineMediaPlayback = true
+
+        webView = WKWebView(frame: self.viewForWebView.bounds, configuration: webConfiguration)
+        webView?.uiDelegate = self
+        webView?.navigationDelegate = self
+        webView?.translatesAutoresizingMaskIntoConstraints = false
+        if let webView = webView {
+            viewForWebView.addSubview(webView)
+        }
+
+        viewForWebView.addConstraint(NSLayoutConstraint(item: webView, attribute: .leading, relatedBy: .equal, toItem: viewForWebView, attribute: .leading, multiplier: 1.0, constant: 0.0))
```

Comment:
Your changes look very nice! Good work. Here you have a Xcode warning "Expression implicitly coerced from 'WKWebView?' to 'Any'". I would create temporary webView in line 49 and assigned it to the webView like: 

`let webView = WKWebView(frame: self.viewForWebView.bounds, configuration: webConfiguration)
self.webView = webView`

Reply:
Thanks! That's a really good way!

Reply:
I have committed this change, Please review it.

Reply:
Please squash and rebase your branch, that Jenkins can test your branch including all other changes. To be sure everything is working.
Thanks.

Reply:
Done!

Reply:
I think you forgot to rebase with the upstream/develop?

Reply:
Yeah. Seems like finally it's done. Thanks!